lib/compile/bundles/bootstrap.js

```diff
@@ -0,0 +1,21 @@
+import most from "most";
+
+import resolveAsset from "../modules/resolve";
+import loadAst from "../modules/load-ast";
+
+export default function (compilation) {
+  return most
+    .from(compilation.opts.emit)
+    .filter(def => def.entry || def.split)
+    .map(function bootstrapBundle (bundleDef) {
+      const src = bundleDef.entry || bundleDef.split;
+      const asset = resolveAsset.call(compilation, src);
```

Comment:
I'm still not crazy about modifying the context of a function as opposed to acting on an object. If nothing else it makes for some less ideal developer ergonomics (if anything here was to be used/understood in isolation). I remember vaguely from before that there were a few places that were made easier by doing this. Now that you're using streams is this still the case?